group_vars/all

```diff
@@ -52,10 +52,12 @@ mandatory_packages:
   - sudo
   - git       # this is a mandatory package since we rely on ansible-pull
   - timelimit # this contains timelimit command which is used to avoid ansible-pull to be too long
+  - nano      # this helps ad-hoc debugging
```

Comment:
wait what? you don't know vi? :)

Reply:
I know vi only enough to do `:wq` after launching it accidentally :)